cdt_slugs.yaml

```diff
@@ -72,11 +72,19 @@ libnl-devel:
   license_file: licenses/libnl-devel-license
 # libpng-devel:
 #   custom: false
-# librdmacm:
-#   custom: false
-# librdmacm-devel:
-#   # generates librdmacm too
-#   custom: false
+librdmacm:
+  custom: false
+  skipped_cdts:
+    - cos7-x86_64
+    - cos7-aarch64
+    - cos7-ppc64le
+librdmacm-devel:
+  custom: false
+  skipped_cdts:
+    - cos7-x86_64
+    - cos7-aarch64
+    - cos7-ppc64le
```

Comment:
Ditto for being skipped here @jakirkham

Reply:
Thanks Matt. Doing in PR ( https://github.com/conda-forge/cdt-builds/pull/26 ) 🙂